zkevm-circuits/src/copy_circuit.rs

```diff
@@ -641,20 +645,10 @@ impl<F: Field> SubCircuitConfig<F> for CopyCircuitConfig<F> {
                         "value_acc(2) == value_acc(0) * r + value(2)",
                         meta.query_advice(value_acc, Rotation(2)),
                         meta.query_advice(value_acc, Rotation::cur()) * challenges.keccak_input()
-                            + meta.query_advice(value, Rotation(2)),
+                            + value_or_pad_2,
                     );
                 },
             );
-            cb.condition(not::expr(meta.query_advice(mask, Rotation::cur())), |cb| {
-                cb.require_zero(
```

Comment:
when is pad & not mask, the value should be zero.  the `value_or_pad_2` is equivalent to this constraint ?